### ISSUE TITLE
Failed to decode file with music_directory using nfs to Ubuntu NFS Share for Win32 version 0.21 and higher 

### ISSUE THREAD

**Issue by Pharmster** [MusicPlayerDaemon/MPD#779]
### Describe the bug
Win32 mpd version 0.21 and higher - In Windows 7 Home Premium, when music_directory is set using protocol nfs to connect to Ubuntu 16.10 NFS share directory. mpd is able to connect to nfs share but failed to play music file with message "Failed to decode music files"

I tested version lower then 0.21 worked successfully and mpd will play music file without issue.

## Expected Behavior
mpd to play music file read from nfs share

## Actual Behavior
mpd failed to play file and display in log and client message "failed to decode file"

## Version

Win32 0.21 and higher 

## Log

Mar 05 21:04 : exception: avformat_open_input() failed: Invalid argument
Mar 05 21:04 : exception: Failed to decode 3 Doors Down/11 So I Need You.mp3
Mar 05 21:04 : player: played "3 Doors Down/11 So I Need You.mp3"

**Comment by MaxKellermann** [comment]
Does this work with local files instead of NFS? The error message looks like it has little to do with NFS.

**Comment by Pharmster** [comment]
Yes, versions 0.21 and higher plays music file locally with no problem or files from windows network shared drive. When I tried to play files on nfs share then I get the error message. Older versions play files from nfs share without any issue.

**Comment by MaxKellermann** [comment]
I suppose this was fixed by 3040ddb5ec51c92c8f840f0500f64f0e0906301e

**Comment by Pharmster** [comment]
I am sorry but don’t know what 3040 ddb means. Do I download the latest Win32 mpd and test if this is resolved or do I need to do something with 3040ddb?

**Comment by MaxKellermann** [comment]
That's technical. Click on the link, that's a code commit id. On that page, you see it's part of "master" (bleeding edge of development) and "v0.21.23" (the latest release you can download and try).

**Comment by Pharmster** [comment]
I apologized for not replying sooner. I tested mpd version .2123 this morning and it is failing with the same error. Please see the below logs entries for .2123 and .2023
I want to install a complete new windows 7 x64 and test this again to make sure it is not my setup. I will send updates.

----------------------------------- mpd.log version .2023
Apr 28 10:47 : client: [0] opened from 192.168.1.92:62297
Apr 28 10:47 : ffmpeg/mp3: Estimating duration from bitrate, this may be inaccurate
Apr 28 10:50 : player: played "U2/U2 - The B-Sides 1980-1990/02. Spanish Eyes.mp3"


------------ mpd.log entries for version .2123
Apr 28 11:01 : client: [1] opened from 192.168.1.92:62332
Apr 28 11:01 : exception: avformat_open_input() failed: Invalid data found when processing input
Apr 28 11:01 : exception: Failed to decode America/America Greatest Hits/A Horse With No Name.flac
Apr 28 11:01 : player: played "America/America Greatest Hits/A Horse With No Name.flac"
Apr 28 11:01 : ffmpeg/mp3: Format mp3 detected only with low score of 1, misdetection possible!
Apr 28 11:01 : ffmpeg/mp3: Failed to read frame size: Could not seek to 1026.
Apr 28 11:01 : exception: avformat_open_input() failed: Invalid argument
Apr 28 11:01 : exception: Failed to decode U2/U2 - The B-Sides 1980-1990/02. Spanish Eyes.mp3
Apr 28 11:01 : player: played "U2/U2 - The B-Sides 1980-1990/02. Spanish Eyes.mp3"
Apr 28 11:01 : ffmpeg/mp3: Format mp3 detected only with low score of 1, misdetection possible!
Apr 28 11:01 : ffmpeg/mp3: Failed to read frame size: Could not seek to 1026.
Apr 28 11:01 : exception: avformat_open_input() failed: Invalid argument
Apr 28 11:01 : exception: Failed to decode 3 Doors Down/01 Kryptonite.mp3
Apr 28 11:01 : player: played "3 Doors Down/01 Kryptonite.mp3"
Apr 28 11:01 : ffmpeg/mp3: Format mp3 detected only with low score of 1, misdetection possible!
Apr 28 11:01 : ffmpeg/mp3: Failed to read frame size: Could not seek to 1026.
Apr 28 11:01 : exception: avformat_open_input() failed: Invalid argument
Apr 28 11:01 : exception: Failed to decode 3 Doors Down/01 Kryptonite.mp3Apr 28 11:01 : player: played "3 Doors Down/01 Kryptonite.mp3"  


    On Friday, April 24, 2020, 12:21:20 AM CDT, Max Kellermann <notifications@github.com> wrote:  
 
 


That's technical. Click on the link, that's a code commit id. On that page, you see it's part of "master" (bleeding edge of development) and "v0.21.23" (the latest release you can download and try).

**Comment by MaxKellermann** [comment]
No update.
